services/provider/server/server_test.go

```diff
@@ -138,7 +139,7 @@ var (
 
 func TestGetExternalResources(t *testing.T) {
 	ctx := context.TODO()
-	objects := []runtime.Object{
+	objects := []crClient.Object{
```

Comment:
Why do we need the transition from `apimachinery client` to `controller-runtime client`?